### ISSUE TITLE
fix bad resize on serial console

### ISSUE THREAD

**Issue by Liambeguin** [Dronolab/meta-drono#36]
when using a serial console, weird stuff happens with less and systemd commands ..

**Comment by Liambeguin** [comment]
```
Unlike ssh, serial connection does not have a mechanism to transfer something 
like SIGWINCH when a terminal is resized. This will cause weird problem with 
some full-screen programs (e.g. less) when you resized your terminal emulator's 
window.
```

**Comment by Liambeguin** [comment]
to my greatest surprise, this was easily fixed with f867ecf2e3a06a9bc925c99a271694c35bd5ff56
We still have an issue if you resize the window after logging in but 
according to the first comment, this is normal and you just have to run ```eval$(resize)```
